xbmc/platform/posix/main.cpp

```diff
@@ -58,7 +59,7 @@ int main(int argc, char* argv[])
   struct rlimit rlim;
   rlim.rlim_cur = rlim.rlim_max = RLIM_INFINITY;
   if (setrlimit(RLIMIT_CORE, &rlim) == -1)
-    CLog::Log(LOGDEBUG, "Failed to set core size limit (%s)", strerror(errno));
+    fprintf(stderr, "Failed to set core size limit (%s)", strerror(errno));
```

Comment:
it's pretty much the same thing but the header is already include for `CMessagePrinter` so you could just use `CMessagePrinter::DisplayError` 😸 It's up to you really.

If you decide to keep this it would be good to add a carriage return `\n`

Reply:
Fixed by adding `\n`